src/packagedcode/__init__.py

```diff
@@ -18,6 +18,7 @@
 from packagedcode import debian
 from packagedcode import debian_copyright
 from packagedcode import distro
+from packagedcode import dockerfile
```

Comment:
You need to change imports after you change filenames :) 

Please run a subset of the test suite locally (atleast the part where you made changes) after you make changes, look for failing CI like : https://dev.azure.com/nexB/scancode-toolkit/_build/results?buildId=16107&view=logs&j=1412dbfa-421a-5a40-8ddf-7073fc46aa19&t=13c90161-2109-5957-f563-a96fefc7d869 and make sure we pass all the tests.